### PR TITLE
Extract Legacy

### PR DESCRIPTION
**Pull request by Drapegnik**
## new commands
```bash
npm run start:legacy
npm run build:legacy
```

## deployed
https://babajka.github.io/babajka-markup/legacy/main-page/main-page.html
https://babajka.github.io/babajka-markup/storybook

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -17,29 +17,32 @@ const ejs = require('gulp-ejs');
...
-
 const config = {
   libsPath: 'node_modules',
   buildPath: 'dist',
   stylesPath: 'styles',
   fontsPath: 'fonts',
   imagesPath: 'images',
   landingPath: 'landing',
-  srcPath: 'src',
+  srcPath: process.env.BABAJKA_LAGACY ? 'legacy' : 'src',
```
Comment (UladBohdan) [inline gulpfile.js]: BABAJKA_L**E**GACY

File: package.json [hunk 2]
```diff
@@ -3,11 +3,15 @@
   "version": "1.1.2",
   "description": "Markups",
   "scripts": {
-    "start": "cross-env PORT=3001 npm run watch",
-    "build": "gulp build && npm run storybook:build",
+    "clean": "rm -rf dist",
+    "start": "npm run clean && cross-env PORT=3001 npm run watch",
+    "start:legacy": "cross-env BABAJKA_LAGACY=true npm start",
```
Comment (UladBohdan) [inline package.json]: here and below: `BABAJKA_LEGACY`

File: src/index.scss [hunk 3]
```diff
@@ -1,12 +1 @@
...
-@import 'bulma';
-// Import bulma-badge
-@import 'bulma-badge';
-// Import FontAwesome
-@import 'font-awesome';
-
-// Fonts.
-@import url('https://fonts.googleapis.com/css?family=Comfortaa|Cormorant');
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&subset=cyrillic');
+// Brave New World!
```
Comment (UladBohdan) [inline src/index.scss]: Did you read?

File: gulpfile.js (hunk 1 above)
Comment (UladBohdan) [reply gulpfile.js]: and I am now sure if that is a good approach.
Switching between `legacy` and `src` means we either build `src`+`landing`+`stubs` or `legacy`+`landing`+`stubs`. But for now it's fine to have everything built e.g. for github deployment: I think we want legacy Wir to be on github pages for some time.
Probably we are fine with building just everything right now and then we'll consider not building legacy code.

Comment (UladBohdan) [reply gulpfile.js]: I see you have separate deployment commands: `deploy:legacy` and `deploy`. But that means we either have old Wir or new Wir on github pages, not both together, right?

File: src/index.scss (hunk 3 above)
Comment (Drapegnik) [reply src/index.scss]: Yep)

File: gulpfile.js (hunk 1 above)
Comment (Drapegnik) [reply gulpfile.js]: fixed

**Pull request merged**